.github/workflows/ci.yml

```diff
@@ -16,6 +16,41 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ steps.get-versions.outputs.node }}
+          cache: yarn
+      - run: yarn install --frozen-lockfile --ignore-scripts
+      - run: yarn build --scope @mux-elements/*
```

Comment:
I think it would be good to run the build on the example projects too, perhaps in parallel. It has caught type issues for us previously related to props usage etc.